crates/stc_ts_type_checker/tests/conformance/types/mapped/mappedTypeRelationships.ts.stats.rust-debug

```diff
@@ -1,6 +1,6 @@
 Stats {
-    required_error: 27,
-    matched_error: 1,
-    extra_error: 25,
-    panic: 0,
```

Comment:
`IndexedAccessType` cause overflow,
if you want decrase panic, you should overflow narrow any